objects/television.rb

```diff
@@ -0,0 +1,122 @@
+#! /usr/bin/env ruby
```

Comment:
No space between `#!` and `/usr/bin...`

Reply:
Gah, I keep doing that!  Don't know why, I don't do it in Perl!